content/articles/data-entry-support-with-html/index.md

```diff
@@ -0,0 +1,81 @@
+    <label>Secret Code: <input type="password"></label>
+    <input type="submit" value="ENTER">
+</form>
+```
+
+### Sending form to Google Sheets
+Google Forms may be utilized as an application by going to the responses button and reading all responses. Whenever you're gathering a bunch of data from some individuals, though, you should put it all in a spreadsheet so you can organize and evaluate it later.
+It takes a long time to transfer information from Google Forms to Google Sheets manually. Alternatively, you may automate data transmission by connecting Google Forms to Google Sheets. It implies that when a respondent uploads a form, the data will display in the spreadsheet 
+simultaneously. 
+
+Follow the procedure below to send the form you created to Google Spreadsheet. It can be either a brand-new or already created spreadsheet.
+1. Select the `response` option in the selected form.
+2. Besides the Google Sheets symbol, select the dots box. Pick Response Location from the drop-down menu.
+3. You may choose where your reply will go from here. You have two options: creating a blank spreadsheet or selecting an already made one.
+4. To see if Google Forms and the spreadsheet have correctly linked, tap the Google Sheets button, which directs you straight to your spreadsheet containing the form replies.
+
+### Deploying a script as a web app
+
+To convert Google scripts to web apps, Google App Script is used to perform this operation. The following procedures must be undertaken to convert a script into a web app.
+* Pick the file and manage versions by creating a new version and saving it. 
+* Choose the Publish option, then deploy it as a web app from the drop-down menu.
+* Click on the file you previously saved beneath the Project version.
+* Choose whose permission with which the app should run under, as your user account or the account of the other users accessing the web app.
+* Select who needs to be accredited to have access to the app. 
+* Tap the Deploy button for the operation to be performed.
+
+Users may execute Google Apps Scripts straight from the browsers if they post them as a web app. For example, an already uploaded Google Form file may publish the script as an open web app, allowing individuals to submit files to your Google Drive without logging into their Google accounts.
+
+A user may select whoever has permission to access the web app after the Google Apps Script has been deployed as a web app. The following are the kinds of permissions used. 
+1. User only: The script creator is the only one who has permission to access the web.
+2. Domain users: The web app is simply accessible to individuals within the Google Apps domain.
+3. Everyone: Anyone has access to and can use the web app. However, the user must first sign in with their Google account.
+4. The last kind of permission allows the web app to be accessed by anybody, even unknown people who do not own a Google account.
+
+### Conclusion
```

Comment:
At the end of your conclusion, add another section "Further reading" and provide links of related articles for your readers.